.schema/config.schema.json

```diff
@@ -1064,6 +1104,17 @@
             }
           }
         },
+        "device_authorization": {
+          "token_polling_interval": {
+            "description": "Sets the starting token polling interval.",
+            "default": "5s",
+            "allOf": [
+              {
+                "$ref": "#/definitions/duration"
+              }
+            ]
+          }
```

Comment:
What is currently the number of chars for the user code? Should we make it configurable? I would vote for 8 characters because 6 is just too likely to be attacked.

Reply:
Currently we use 8 upper case characters.

Reply:
thanks!